docs/streams/tutorial.html

```diff
@@ -62,7 +62,7 @@
     </pre>
 
     <p>
-        <strong>Important:</strong> You must manually update the setting of <code>&lt;kafka.version&gt;</code> in the generated <code>pom.xml</code> file from <code>0.11.0.1-SNAPSHOT</code> to <code>0.11.0.1</code>.
+        <strong>Important:</strong> You must manually update the setting of <code>&lt;kafka.version&gt;</code> in the generated <code>pom.xml</code> file from <code>{{fullDotVersion}}-SNAPSHOT</code> to <code>{{fullDotVersion}}</code>.
         <strong>Note:</strong> in the next release the above step will not be required.
```

Comment:
@guozhangwang is this step still needed in 0.11.0.2?

Reply:
As long as we now follow the release process instructions when releasing to remove the `-SNAPSHOT` suffix, it should be good. Previously in 0.11.0.1 we forgot to remove the suffix when uploading and hence added this.

We can remove this line.

Reply:
@guozhangwang Thank you, I have removed the line. Can you review please? Thanks.